scarb/tests/build_cairo_plugin.rs

```diff
@@ -1573,3 +1574,287 @@ fn can_expand_impl_inner_func_attrr() {
 
         "#});
 }
+
+#[test]
+fn can_use_quote() {
+    let temp = TempDir::new().unwrap();
+    let t = temp.child("some");
+    let cairo_quote_path = CAIRO_LANG_QUOTE_PATH.to_string();
+    CairoPluginProjectBuilder::default()
+        .add_dep(formatdoc! {r#"cairo-lang-quote = {{ path={cairo_quote_path} }}"#})
+        .add_dep(r#"cairo-lang-primitive-token = "1.0.0""#)
+        .lib_rs(indoc! {r##"
+        use cairo_lang_macro::{ProcMacroResult, TokenStream, inline_macro};
+        use cairo_lang_quote::quote;
+
+        #[inline_macro]
+        pub fn some(_token_stream: TokenStream) -> ProcMacroResult {
+                let tokens = quote! {
+                    5
+                };
+                ProcMacroResult::new(tokens)
+        }
+        "##})
+        .build(&t);
+    let project = temp.child("hello");
+    ProjectBuilder::start()
+        .name("hello")
+        .version("1.0.0")
+        .dep("some", &t)
+        .lib_cairo(indoc! {r#"
+            fn main() -> felt252 { some!() }
+        "#})
+        .build(&project);
+
+    Scarb::quick_snapbox()
+        .arg("cairo-run")
+        // Disable output from Cargo.
+        .env("CARGO_TERM_QUIET", "true")
+        .current_dir(&project)
+        .assert()
+        .success();
```

Comment:
Can we add assert on returned value, to make sure it is 5 indeed?

Reply:
I did assert the result on 2 last tests, as they are using the attribute macro. As it turns out, the `scarb expand` doesn't really work with inline macros

Reply:
In those inline macro tests, we test for the returned value and its type, as the `cairo-run` would throw an error, when the main function haven't returned the proper type